raiden/api/rest.py

```diff
@@ -84,8 +95,8 @@ def add_resource(self, resource_cls):
             resource_cls._route
```

Comment:
`flask-restful` has a facility for [passing constructor arguments](http://flask-restful.readthedocs.io/en/0.3.5/intermediate-usage.html#passing-constructor-parameters-into-resources) to the resource, that way we can properly construct the resource instead of having a partially initialized object and setting the `rest_api` attribute afterwards.